ClientApp/viking-vault/src/UserDashboard/TransactionList.tsx

```diff
@@ -153,11 +153,11 @@ class TransactionList extends React.Component<any, IState> {
     private getTableRowFromTransaction(tran: ITransaction) {
         return <tr>
             <td><img src={this.getImageForTransactionType(tran.type)} className={"other-party-image"}/></td>
-            <td className="font-weight-bold">{this.formatOtherPartyString(tran)}</td>
+            <td className="font-weight-bold details" title={this.formatOtherPartyString(tran)}>{this.formatOtherPartyString(tran)}</td>
```

Comment:
you could extract {this.formatOtherPartyString(tran)} and reuse it in both calls